SimPEG/Maps.py

```diff
@@ -759,16 +759,30 @@ class PolyMap(IdentityMap):
 
             m = [\sigma_1, \sigma_2, c]
 
+        Can take in an actInd vector to account for topography.
+
     """
-    def __init__(self, mesh, order, logSigma=True, normal='X'):
+    def __init__(self, mesh, order, logSigma=True, normal='X', actInd = None):
```

Comment:
can we change `actInd` to `indActive` to be consistent with the regularization, etc?